subsys/bootloader/bootloader.c

```diff
@@ -85,6 +85,16 @@ void uninit_used_peripherals(void)
 #endif
 }
 
+#ifdef CONFIG_SW_VECTOR_RELAY
+#ifndef CONFIG_SB_C_RUNTIME_SETUP_VARIANT_ZEPHYR
+_GENERIC_SECTION(.vt_pointer_section) u32_t _vector_table_pointer;
+#endif
+extern u32_t _vector_table_pointer;
+#define _VTOR _vector_table_pointer
```

Comment:
Why not simply `VTOR`? It's confusing to prefix symbols with underscores.. are they linker generated? are they compiler intrinsicts? are they "private"?

Reply:
Good point, I'll change it. This was left from a previous iteration.

Reply:
Done